build/yamls/base/crds.yml

```diff
@@ -100,6 +102,10 @@ spec:
       jsonPath: .status.egressNode
       name: Node
       type: string
+    - description: The Bandwidth limit of the selected workloads traffic.
+      jsonPath: .spec.bandwidth
+      name: Bandwidth
```

Comment:
Should we name it BandwidthLimit to be clear?

Reply:
Maybe `Bandwidth ` also express relevant meaning, and  it is simpler.